### PR TITLE
Update details for make-app and code-a-thon

### PR DESCRIPTION
**Pull request by pradiptikayal**

### CODE REVIEW

File: src/content/_data/events.yaml [hunk 1]
```diff
@@ -99,10 +99,10 @@ make-app:
   teamSize: 2
   lang: cpp
   rounds:
-    - Elimination round (Multiple-choice based questions based on concepts of Java/Android).
-    - |-
-      The qualifying teams would be asked to develop an android/ios application overnight at their own places and next day have to present their project to judges for final evaluation
+    -  The teams would be asked to develop an android/ios application on spot at day 2 of Sankalan. Theme will be provided on spot.They will be given 8 hours that is from 10am-6pm. They have to present their project to judges for final evaluation
```
Comment (sidvishnoi) [inline src/content/_data/events.yaml]: just keep first sentence in rounds.
and move rest of them to details, as separate points.

Comment (pradiptikayal) [reply src/content/_data/events.yaml]: okk

**Comment by pradiptikayal** [comment]
Please check it once.

**Comment by sidvishnoi** [comment]
please create another pull request. i rebased it wrong

**Pull request merged**